dispatcher.yml

```diff
@@ -1,20 +1,23 @@
 # Demo config
 ---
-pool:
-  max_workers: 3
-producers:
-  brokers:
-    # List of channels to listen on
+service:
+  max_workers: 4
+brokers:
+  pg_notify:
+    config:
+      conninfo: dbname=dispatch_db user=dispatch password=dispatching host=localhost port=55777
+    sync_connection_factory: dispatcher.brokers.pg_notify.connection_saver
     channels:
     - test_channel
     - test_channel2
     - test_channel3
-    pg_notify:
-      # Database connection details
-      conninfo: dbname=dispatch_db user=dispatch password=dispatching host=localhost
-        port=55777
-  scheduled:
-    'lambda: __import__("time").sleep(1)':
-      schedule: 3
-    'lambda: __import__("time").sleep(2)':
-      schedule: 3
+    default_publish_channel: test_channel
+producers:
+  ScheduledProducer:
+    task_schedule:
```

Comment:
I was wondering that scheduled tasks are usually more dependent on the logic of the application and as any other task, may be more useful if they can be registered in runtime by the application. Is something that you have considered @AlanCoding  ?

Reply:
Current model is that `@task` can specify "options" for the task, but not triggers for the task. This feels defensible for the time-being, but a single slate of specifications has appeal. It's just that I would do that by moving everything into the config, as opposed to moving everything into the code.